server/src/db/migrations/20230202145724_createReviews.cjs

```diff
@@ -0,0 +1,24 @@
+/**
+ * @typedef {import("knex")} Knex
+ */
+
+/**
+ * @param {Knex} knex
+ */
+exports.up = async (knex) => {
+    return knex.schema.createTable("reviews", (table) => {
+        table.bigIncrements("id")
+        table.string("review")
```

Comment:
I know this will be quite a change, but its not ideal to have a `review` column on a reviews table. It leads to `review.review` syntax which makes it more difficult to know what `review` specifically means. `content` or `body` would be good alternatives to fix this with

Reply:
Changed from review to content